### PR TITLE
M2-5316: [Admin Panel] Logging Raw Responses (with hidden Items) for QA in debugMode

### PR DESCRIPTION
**Pull request by alex-i-e**
[M2-5316](https://mindlogger.atlassian.net/browse/M2-5316)

[M2-5316]: https://mindlogger.atlassian.net/browse/M2-5316?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/Dashboard/hooks/useDecryptedActivityData.ts [hunk 1]
```diff
@@ -85,6 +88,17 @@ export const useDecryptedActivityData = (
       }
     }
 
+    checkIfShouldLogging() &&
+      logDataInDebugMode({
```
Comment (nmialik) [inline src/modules/Dashboard/hooks/useDecryptedActivityData.ts]: it seems `logDataInDebugMode` already has a check for `checkIfShouldLogging`?
`export const logDataInDebugMode = (data: unknown) => checkIfShouldLogging() && console.log(data);`

Comment (alex-i-e) [reply src/modules/Dashboard/hooks/useDecryptedActivityData.ts]: Yes, that's right! I'll get rid of it.

**Review by nmialik — APPROVED** [review]

File: src/modules/Dashboard/hooks/useDecryptedActivityData.ts [hunk 2]
```diff
@@ -85,6 +87,22 @@ export const useDecryptedActivityData = (
...
     }
 
+    /*
+    Here we use 'any' because of differences in API responses when Export Data
+    and in Datawiz.
+    Thus, in dataviz summary the responses are retrived without details about:
+    'version', 'client.appId', 'client.appVersion'
+    */
+    logDataInDebugMode({
+      [`applet_v.${(rest as any as ExportAnswer)?.version ?? ''}/appId_${
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/hooks/useDecryptedActivityData.ts]: Can we use `rest as unknown as ExportAnswer`?

Comment (alex-i-e) [reply src/modules/Dashboard/hooks/useDecryptedActivityData.ts]: agree

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**